cardano-api/src/Cardano/Api/IPC.hs

```diff
@@ -209,20 +222,15 @@ mkVersionedProtocols :: forall block.
                         )
                      => NetworkId
                      -> ProtocolClientInfoArgs block
-                     -> LocalNodeClientProtocolsForBlock block
+                     -> (NodeToClientVersion -> LocalNodeClientProtocolsForBlock block)
                      -> Net.Versions
                           Net.NodeToClientVersion
                           Net.NodeToClientVersionData
                           (Net.OuroborosApplication
                              Net.InitiatorMode
                              Net.LocalAddress
                              LBS.ByteString IO () Void)
-mkVersionedProtocols networkid ptcl
-                     LocalNodeClientProtocolsForBlock {
-                       localChainSyncClientForBlock,
-                       localTxSubmissionClientForBlock,
-                       localStateQueryClientForBlock
-                     } =
+mkVersionedProtocols networkid ptcl unversionedClients =
```

Comment:
I'm not sure I quite understand the variable name `unversionedClients`, but not a big deal.